zh-cn/json-cn.html.markdown

```diff
@@ -8,24 +8,30 @@ filename: learnjson-cn.json
 lang: zh-cn
 ---
 
-因为JSON是一个极其简单的数据交换格式，本教程最有可能成为有史以来最简单的
-Learn X in Y Minutes。
+JSON是一个极其简单的数据交换格式。按[json.org](https://json.org)说的，它对人类易读易写，对机器易解析易生成。
 
-纯正的JSON实际上没有注释，但是大多数解析器都
-接受C-风格(//, /\* \*/)的注释。为了兼容性，最好不要在其中写这样形式的注释。
+一段JSON必须按以下两种方式之一呈现：
+
+* 一个键值对的集合(`{ }`)。按不同语言，这可能被理解为对象/记录/结构体/字典/哈希表/有键列表/关联数组
+* 一个有序的值列表(`[ ]`)。按不同语言，这可能被理解为数组/向量/列表/序列
```

Comment:
This is incorrect (an it seems that the English version is wrong as well). According to the grammar here: https://www.crockford.com/mckeeman.html which is linked from json.org, any single value is also valid json. The English original of these lines seem to be a misread of the explanation on json.org. The primary source intends to explain the two types of *structures* of JSON, i.e. objects and arrays, but does not preclude having single values as valid JSON.

Reply:
It can also be inferred from json.org.
`json` contains `element`. `element` contains `ws value ws`. `value` contains `object array string number true false null`.
So a single number is valid.